### ISSUE TITLE
pytorch version

### ISSUE THREAD

**Issue by HOUYONGKUO** [zgojcic/3DSmoothNet#42]
HI,
can you share code ( in pytorch ) with me?
3Q.

**Comment by zgojcic** [comment]
Hi all, the pytorch reimplementation is available under [code](https://share.phys.ethz.ch/~gsg/3DSmoothNet/pytorch_reimplementation/) and [training data](https://share.phys.ethz.ch/~gsg/3DSmoothNet/pytorch_training_data/). Note that the model is slightly different, e.g. no dropout and was not extensively tested. There might be bugs or some other problems.
